install/kubernetes/helm/istio/README.md

```diff
@@ -116,6 +116,11 @@ The chart deploys pods that consume minimum resources as specified in the resour
     EOF
     ```
 
+1. Add `istio.io` chart repository and point to the release:
+    ```
+    $ helm repo add istio.io https://storage.googleapis.com/istio-prerelease/daily-build/release-1.1-latest-daily/charts
```

Comment:
What will be the URL for production users ? We can't use prerelease and storage.

Reply:
it will be replaced during creating release package.
```
  if [ "${CB_PIPELINE_TYPE}" = "monthly" ]; then
    local origin_url="istio-prerelease/daily-build/master-latest-daily/charts"
    local target_url="istio-release/releases/${VER_STRING}/charts"
    sed -i.bak "s:${origin_url}:${target_url}:g" "${COMMON_FILES_DIR}/install/kubernetes/helm/istio/README.md"
    rm -rf "${COMMON_FILES_DIR}/install/kubernetes/helm/istio/README.md.bak"
  fi
}
```

Reply:
@clyang82 Can you please add some comments to production users here? +1 to @costinm , it seems to be weird to use pre release here.